source/guides/rapids-docker-with-cuda.md

```diff
@@ -0,0 +1,371 @@
+
+# Install Base CUDA Components (from base image)
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    cuda-cudart-12-9=${NV_CUDA_CUDART_VERSION} \
+    cuda-compat-12-9 && \
+    rm -rf /var/lib/apt/lists/*
+
+# CUDA Environment Configuration
+ENV PATH=/usr/local/cuda/bin:${PATH}
+ENV LD_LIBRARY_PATH=/usr/local/cuda/lib64
+
+# NVIDIA Container Runtime Configuration
+ENV NVIDIA_VISIBLE_DEVICES=all
+ENV NVIDIA_DRIVER_CAPABILITIES=compute,utility
+
+# Required for nvidia-docker v1
+RUN echo "/usr/local/cuda/lib64" >> /etc/ld.so.conf.d/nvidia.conf
+
+# Install system dependencies
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    wget \
+    curl \
+    git \
+    ca-certificates \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install Miniforge
+RUN wget -qO /tmp/miniforge.sh "https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh" && \
+    bash /tmp/miniforge.sh -b -p /opt/conda && \
+    rm /tmp/miniforge.sh && \
+    /opt/conda/bin/conda clean --all --yes
+
+# Add conda to PATH
+ENV PATH="/opt/conda/bin:${PATH}"
+
+# Create conda group and rapids user
+RUN groupadd -g 1001 conda && \
+    useradd -rm -d /home/rapids -s /bin/bash -g conda -u 1001 rapids && \
+    chown -R rapids:conda /opt/conda
+
+USER rapids
+WORKDIR /home/rapids
+
+# Copy the environment file template
+COPY env.yaml /home/rapids/env.yaml
+
+# Update the base environment with user's packages from env.yaml
```

Comment:
I remember we chatted about this, should we create a new environment instead of updating base? This is what was done on other docs but because it was a requirement, or we are doing the same in the rapids/jupyter images?